compiler/src/dotty/tools/dotc/core/NameKinds.scala

```diff
@@ -320,7 +320,7 @@ object NameKinds {
   val PatMatStdBinderName: UniqueNameKind    = new UniqueNameKind("x")
   val PatMatAltsName: UniqueNameKind         = new UniqueNameKind("matchAlts")
   val PatMatResultName: UniqueNameKind       = new UniqueNameKind("matchResult")
-  val PatMatVarName: UniqueNameKind          = new UniqueNameKind("ev$")
+  val PatMatVarName: UniqueNameKind          = new UniqueNameKind("$given")
```

Comment:
If this is only used for givens, maybe this should be PatMatGivenName?

Reply:
Technically it is only used to to be able to identify a synthetic name as a pattern variable name.

Reply:
But it is true that if it is synthetic, the only way to access it is if it is given. I will rename it.